dask/utils.py

```diff
@@ -558,6 +558,18 @@ def funcname(func):
         return str(func)
 
 
+def typename(typ):
+    """
+    Return the name of a type
+
+    Examples
+    --------
+    >>> typename(int)
+    'builtins.int'
+    """
+    return typ.__module__ + '.' + typ.__name__
```

Comment:
Recommendations on how to do this better would be welcome

Reply:
Supposedly `__module__` can be `None` (see [docs](https://docs.python.org/3/reference/datamodel.html)), although I've yet to find a case where it is. You also may want to drop the module name for builtins.

```suggestion
    m = typ.__module__
    n = typ.__name__
    return n if m in {None, 'builtins'} else '%s.%s' % (m, n)
```